package.json

```diff
@@ -30,7 +30,7 @@
     "@types/auto-launch": "^5.0.0",
     "@types/cordova-plugin-device": "^0.0.3",
     "@types/jasmine": "^2.8.6",
-    "@types/node": "^8.0.53",
+    "@types/node": "^14.14.7",
```

Comment:
Do we need to stay in sync with the version of Node that Electron 11 uses (v12.18.3)?

Reply:
In theory, we should since our code runs in node v12. In practice, there doesn't seem to be an issue with us depending on v14 types. I believe electron 12 (currently in beta) depends on node 14, so hopefully the mismatch will be temporary.